app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
 
-  def show; end
```

Comment:
- [ ] The same thing goes here also make sure that the posts are rendering on the user page.

Reply:
I have this code in my show method:
`  def show
    @post = Post.find(params[:id])
  end`
  and I can see it is rendering the data for a specific Post.